azure-db/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG PG_IMAGE=postgres:11-alpine
+
+FROM ${PG_IMAGE}
```

Comment:
Why `postgres:11-alpine` is a variable ?

Reply:
unlikely but we could build the images for different PG versions.

Reply:
(important for backups)